src/data-dictionary/IgSummary.ts

```diff
@@ -252,20 +252,36 @@ export class IgSummary {
     public async generateSpreadsheet() {
         // Get external dependencies
         await this.getExternalDefs();
-
+        
         // Store data for CSV here
         let profileElements: Array<DataElementInformationForSpreadsheet> = [];
-
+        
         // Iterate through StructureDefinitions
         // Note that SUSHI creates duplicate objects for each Profile because it wants to index by URL, ID, and name.
         // `_.uniqBy()` cleans this up.
         if (this.defs.allProfiles().length == 0) {
             logger.error(`No profiles found in ${this.fhirDefFolder}.`);
             return;
         }
+
+        let extensionFlgMap = new Map<string, string>();
+        
         for (const sd of _.uniqBy(this.defs.allProfiles(), 'id')) {
             const snapshot = sd.snapshot;
+            
+            for (const elemJson of snapshot.element.slice(1)) {
+                elemJson.extension?.forEach((ext: any) => {
+                    this.settings.extensionColumn?.forEach((col:string) => {
+                        console.log(sd.title+elemJson.id+col);
```

Comment:
this results in a lot of output - I think it should be removed before merging

Reply:
Oh, forgot to remove it. Done.